Mathlib/LinearAlgebra/FiniteDimensional/Basic.lean

```diff
@@ -393,6 +393,25 @@ theorem ofInjectiveEndo_left_inv (f : V →ₗ[K] V) (h_inj : Injective f) :
     ((ofInjectiveEndo f h_inj).symm : V →ₗ[K] V) * f = 1 :=
   LinearMap.ext <| (ofInjectiveEndo f h_inj).symm_apply_apply
 
+variable {V' : Type*} [AddCommGroup V'] [Module K V'] [FiniteDimensional K V']
+omit [FiniteDimensional K V]
+
+/-- An injective linear map between finite-dimensional modules of equal rank
+is a linear equivalence.
+Unlike `LinearEquiv.ofFinrankEq` (which creates an *abstract* linear equivalence between `V` and `V'`),
```

Comment:
```suggestion
is a linear equivalence.

Unlike `LinearEquiv.ofFinrankEq` (which creates an *abstract* linear equivalence between `V` and `V'`),
```
Since otherwise these lines will be merged in the web docs.